set_symfony_version.sh

```diff
@@ -2,19 +2,19 @@
 
 # to enable git-for-windows symlinks, see here: https://github.com/git-for-windows/git/pull/156
 export MSYS=winsymlinks:nativestrict
-VERSION_DEFAULT="5.4"
+SYMFONY_VERSION_DEFAULT="5.4"
 
 setSymfonyVersion() {
-  VERSION=$1
-  if [[ -z $1 || "$1" == "default" ]]; then
-    VERSION="$VERSION_DEFAULT"
+  [[ -z "${SYMFONY_VERSION}" ]] && SYMFONY_VERSION=$1
+  if [[ -z $SYMFONY_VERSION || "${SYMFONY_VERSION}" == "default" ]]; then
+    SYMFONY_VERSION="${SYMFONY_VERSION_DEFAULT}"
```

Comment:
added support for environmental variable SYMFONY_VERSION
such that you can run:

export SYMFONY_VERSION=6.2

and then call "bash install.sh" 
so no need to add "6.2" as an argument everytime if you want to switch locally